maven-plugin/tests/pom.xml

```diff
@@ -71,16 +70,16 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
```

Comment:
Replace this with a property <maven.deploy.skip>true

Reply:
Done